prow/plugins.yaml

```diff
@@ -211,9 +214,11 @@ plugins:
     - wip
     - trigger
 
-  kubernetes/charts:
+  helm/charts:
```

Comment:
you'll probably want to copy more of this list: https://github.com/kubernetes/test-infra/blob/72e3821fe1baa8332ae8c78e805049033fd8f5a0/prow/plugins.yaml#L378-L396

Reply:
I was just going through that list to make some more additions :)